plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -129,7 +128,41 @@ public void testDropTableBadLocation()
         // break the table by deleting all its files including transaction log
         deleteRecursively(tableLocation, ALLOW_INSECURE);
 
-        // try to drop table
+        // Assert queries fail cleanly
+        assertQueryFails("TABLE " + tableName, "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("SELECT * FROM " + tableName + " WHERE false", "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("SELECT 1 FROM " + tableName + " WHERE false", "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("SHOW CREATE TABLE " + tableName, "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("CREATE TABLE a_new_table (LIKE " + tableName + " EXCLUDING PROPERTIES)", "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("DESCRIBE " + tableName, "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("SHOW COLUMNS FROM " + tableName, "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("SHOW STATS FOR " + tableName, "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("ANALYZE " + tableName, "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("ALTER TABLE " + tableName + " EXECUTE optimize", "Metadata not found in transaction log for tpch.bad_person");
+        assertQueryFails("ALTER TABLE " + tableName + " EXECUTE vacuum", "Metadata not found in transaction log for tpch.bad_person");
+        assertQuerySucceeds("ALTER TABLE " + tableName + " RENAME TO bad_person_some_new_name");
+        assertQuerySucceeds("ALTER TABLE bad_person_some_new_name RENAME TO " + tableName);
+        assertQueryFails("ALTER TABLE " + tableName + " ADD COLUMN foo int", "Metadata not found in transaction log for tpch.bad_person");
+        // TODO (https://github.com/trinodb/trino/issues/16248) ADD field
```

Comment:
https://github.com/trinodb/trino/issues/16248

Reply:
i linked to the PR that adds syntax.
Once we have syntax supported we should have an uncommented test line here, even before the functionality is supported in Delta